lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -148,6 +149,10 @@ def call(params)
             }.compact
           end
 
+          semaphore = Mutex.new
+
+          params.gdc_logger.info("Synchronizing in mode \"#{mode}\"")
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)